stl/inc/atomic

```diff
@@ -680,33 +703,27 @@ struct _Atomic_storage<_Ty, 8> { // lock-free using 8-byte intrinsics
             return;
         }
     }
-#endif // _M_IX86
 
     _NODISCARD _Ty load() const noexcept { // load with sequential consistency
         const auto _Mem = _Atomic_address_as<const long long>(_Storage);

```

Comment:
Your new `_Atomic_address_as` overload copies `const` from the source to the dest, and the 8/16/32-bit `load` implementations rely on this (simply passing `char`/`short`/`int` to `_Atomic_address_as`). Although `const long long` continues to work, I believe it should be changed for consistency - your overload will make this more readable:
```suggestion
        const auto _Mem = _Atomic_address_as<long long>(_Storage);
```